### PR TITLE
Fix crashlytics

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-14113
affects: Student
release note: none

### things that changed:
- step lib should not update Info.plist w/ `Fabric` Info any longer
- changed default version from 6.4 to 0.0.1 (on student) so it's easy to tell in firebase that 0.0.1 is not anything to worry about
- step repo setup must be run for local builds to work
- upload-symbol was changed from Fabric pod to FirebaseCrashlytics pod
- Seems now, only need to import Firebase for all things Firebase

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1026189&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|13
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/ngl62ja) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fngl62ja)
teacher: [Link](https://rebrand.ly/nzxse5h) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fnzxse5h)
parent: [Link](https://rebrand.ly/z5h6kmt) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fz5h6kmt)

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1026191&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|13
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Student

[MBL-14113](https://instructure.atlassian.net/browse/MBL-14113)
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1026192&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|13
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: CanvasCore/CanvasCore/React Native Modules/CanvasCrashlytics.m [hunk 1]
```diff
@@ -96,42 +96,52 @@ - (dispatch_queue_t)methodQueue
...
 RCT_EXPORT_METHOD(setUserEmail:(NSString *)email)
 {
-    [[Crashlytics sharedInstance] setUserEmail:email];
+    //      this has been deprecated in firebase and discouraged
+    //      https://firebase.google.com/docs/crashlytics/upgrade-sdk?authuser=1#switch-pods
+    /*
+     We adopted the method name setUserID to be consistent with other Firebase APIs and removed setUserName and setUserEmail to discourage logging PII through Crashlytics.
+     */
+//    [[Crashlytics sharedInstance] setUserEmail:email];
 }
```
Comment (vanwagonet) [inline CanvasCore/CanvasCore/React Native Modules/CanvasCrashlytics.m]: Can we just remove these two methods on the objc side, and also remove them on the js side (src/common/CanvasCrashlytics.js)? As far as I can tell these aren't used anyway.

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1026194&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|13
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: Student/Student.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -3338,6 +3360,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
+				MARKETING_VERSION = 0.0.1;
```
Comment (vanwagonet) [inline Student/Student.xcodeproj/project.pbxproj]: should we revert these after testing?

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1026220&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|13
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Pull request merged**